clients/vscode-hlasmplugin/proc_grps_schema

```diff
@@ -7,17 +7,32 @@
             "type": "array",
             "items": {
                 "type": "object",
-                "properties" : {
+                "properties": {
                     "name": {
-                        "type" : "string",
-                        "description" : "User-defined name of processor group"
+                        "type": "string",
+                        "description": "User-defined name of processor group"
                     },
                     "libs": {
-                        "type" : "array",
-                        "description" : "List of folders containing external files such as macros and copy.\nAll files inside these folders are being automatically recognized as HLASM.\nIf you would like to use dependency files with a specific extension, create an extension wildcard in pgm_conf.json for it.\nExtension wildcards must follow the format: [anything]*.[1+ non-whitespace characters], e.g. libs/*.hlasm or *.asm",
-                        "items" : {
-                            "type" : "string"
+                        "type": "array",
+                        "description": "List of folders containing external files such as macros and copy.\nAll files inside these folders are being automatically recognized as HLASM.\nIf you would like to use dependency files with a specific extension, create an extension wildcard in pgm_conf.json for it.\nExtension wildcards must follow the format: [anything]*.[1+ non-whitespace characters], e.g. libs/*.hlasm or *.asm",
+                        "items": {
+                            "type": "string"
                         }
+                    },
+                    "asm_options": {
+                        "type": "object",
+                        "description": "List of Assembler Compile",
+                        "properties": {
+                            "SYSPARM": {
+                                "type": "string",
+                                "description": "It specifies the character string the assembler assigns to the &SYSPARM system variable symbol. "
+                            },
+                            "PROFILE": {
+                                "type": "string",
+                                "description": "Profile Member to be copied int the source program."
+                            }
+                        },
+                        "required": [ "SYSPARM", "PROFILE" ]
```

Comment:
I don't think these should be required.

Reply:
I added it thinking if some one adds asm_options(which is optional ) , the user wants to use them.

Reply:
Yes, but for example I've never used the `PROFILE` parameter. So it makes sense to allow not specifying all of them.